services/NotificationAgent/Dockerfile

```diff
@@ -5,7 +5,6 @@ VOLUME ["/etc/iplant/de"]
 
 COPY conf/main/logback.xml /home/iplant/
 COPY target/notificationagent-standalone.jar /home/iplant/
```

Comment:
The entrypoint below, in conjunction with these lines' copying locations, still depends on the `WORKDIR /home/iplant` removed from `javabase` (it adds `.` to the classpath, which is the `/home/iplant` here by way of the `WORKDIR`). They should be copied into whatever directory is the default workdir (`/`, I believe, for `jeanblanchard/java:8` which is the base of `javabase`) or another alternately configured working directory such as `/root`.